src/Storefront/Controller/AuthController.php

```diff
@@ -177,6 +178,10 @@ public function login(Request $request, RequestDataBag $data, SalesChannelContex
             if ($e instanceof CustomerAuthThrottledException) {
                 $waitTime = $e->getWaitTime();
             }
+        } catch (PasswordPoliciesUpdatedException $e) {
+            $this->addFlash(self::WARNING, $this->trans('account.passwordPoliciesUpdated'));
+
+            return $this->forwardToRoute('frontend.account.recover.page');
         }
 
         $data->set('password', null);
```

Comment:
Maybe put that unset of the password into the `finally` block of the `try-catch` construct, as in that case now the request is forwarded with the password still in memory and in the data bag.

Reply:
@aragon999 Thanks for your comment. You're right. I have updated it.